code/src/js components/Questionrange.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import "css components/questionrange.css";
+
+export const Questionrange = ({coffeeprice, onRangeChange}) => {
+
+  return (
```

Comment:
One suggestion is to add a label here, just to improve the user experience ;) Like a scale from 1-5 or something.